firmware/third_party/CMakeLists.txt

```diff
@@ -42,6 +42,18 @@ file(GLOB_RECURSE SBG_ECOM_SRCS
 )
 list(APPEND SHARED_ARM_BINARY_X86_INCOMPATIBLE_THIRD_PARTY_SRCS ${SBG_ECOM_SRCS})
 
+# Import littlefs
+CPMAddPackage(
+  NAME LITTLEFS
+  GITHUB_REPOSITORY littlefs-project/littlefs
+  GIT_TAG v2.8.0
+  SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/_dep/littlefs 
```

Comment:
I think if you don't specifiy the source_dir it autoamatically puts it in this folder.